dogs-initial/src/main/resources/application.properties

```diff
@@ -0,0 +1,8 @@
+
+server.port=${PORT:5000}
+server.servlet.context-path=/doggos
+
+server.error.whitelabel.enabled=false
+
```

Comment:
Line 3 - note that this will require you to prepend `"/doggos"` to all of your your endpoint URI's .